CorsixTH/Lua/app.lua

```diff
@@ -1585,11 +1605,12 @@ function App:afterLoad()
                        ") to " .. new .. " (" .. self:getVersion() ..
                        "). The save was created using " .. first ..
                        " (" .. self:getVersion(first) .. ")")
```

Comment:
Would be nice if you could refactor this to a string.format call, this looks beyond ugly imho